File: Proyecto-Prog_Avanzada_Frontend-master/src/app/modelo/detalle-compra-get-dto.ts
export class DetalleCompraGetDTO {
    codigo: number;
    codigoPublicacion: number;
    unidades: number;
    precioUnidad: number;

    constructor(codigo: number, codigoPublicacion: number, unidades: number, precioUnidad: number) {
        this.codigo = codigo;
        this.codigoPublicacion = codigoPublicacion;
        this.unidades = unidades;
        this.precioUnidad = precioUnidad;
    }
}

File: Proyecto-Prog_Avanzada-railway/src/main/java/co/edu/uniquindio/proyecto/proyectotienda/dto/DetalleCompraGetDTO.java
package co.edu.uniquindio.proyecto.proyectotienda.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

@AllArgsConstructor
@Getter
@Setter
public class DetalleCompraGetDTO {

    private int codigo;
    private int codigoPublicacion;
    private int unidades;
    private double precioUnidad;
}
